sdk/tables/azure-data-tables/README.md

```diff
@@ -105,15 +105,16 @@ Two different clients are provided to interact with the various components of th
 
 ##### Creating and deleting a table
 
-In order to Create/Delete a table we need to create a TablesClient first.
+In order to Create/Delete a table we need to create a `TableServiceClient` first.
 
 ```cpp
 #include <azure/data/tables.hpp>
 ...
 using namespace Azure::Data::Tables;
 const std::string TableName = "sample1";
 ...
-auto tableServiceClient = TableServiceClient::CreateFromConnectionString(GetConnectionString());
+auto credential = std::make_shared<Azure::Identity::DefaultAzureCredential>();
+auto tableServiceClient = Azure::Data::Tables::TableServiceClient(GetServiceUrl(), credential);
```

Comment:
We should start using code snippets for our samples in docs. This way, they remain in sync.

Here's an example of how to do it:
https://github.com/Azure/azure-sdk-for-cpp/pull/6020

Following this doc:
https://github.com/Azure/azure-sdk-for-cpp/blob/main/doc/SnippetGeneration.md#snippet-use-in-documentation

Reply:
Tracked in https://github.com/Azure/azure-sdk-for-cpp/issues/6266